lib/baconmail/authorizer.rb

```diff
@@ -0,0 +1,56 @@
+# require 'google/api_client/client_secrets'
+require 'googleauth/stores/file_token_store'
+
+# encoding: UTF-8
+module Baconmail
+  class Authorizer
+    OOB_URI = 'urn:ietf:wg:oauth:2.0:oob'
+    SCOPE = Google::Apis::GmailV1::AUTH_SCOPE
+
+    class << self
+      def call(user_id)
+        raise "Google Client ID or Google Client Secret are invalid." if invalid_keys?
```

Comment:
When I try to run `bin/baconmail authorize` I get the following error:

```
baconmail/lib/baconmail/authorizer.rb:12:in `call': Google Client ID or Google Client Secret are invalid. (RuntimeError)
```

I think you are checking if the keys are invalid before we fetch them?

Reply:
@caffo hmm, did you add the keys to `.baconmail`? See an example in README: https://github.com/mbirman/baconmail/tree/google-api-client#setup-instructions